src/components/side_panel/find_and_replace/find_and_replace_store.ts

```diff
@@ -46,10 +47,7 @@ export class FindAndReplaceStore extends SpreadsheetStore {
       this.model.dispatch("SET_FORMULA_VISIBILITY", { show: this.initialShowFormulaState });
       this.updateSearchContent.stopDebounce();
     });
-  }
-
-  get renderingLayers() {
-    return ["Search"] as const;
+    get(HighlightStore).register(this);
```

Comment:
We should unregister to avoid memory leak. It may not be necessary here because the store is never used as a local store, but it's a good practice nonetheless to follow, in case this code is taken as an example and copy-pasted